mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -0,0 +1,94 @@
+from azure.storage.blob import BlobClient
```

Comment:
We'll also want to add `azure.storage.blob` as a dependency of MLflow in `setup.py`: https://github.com/mlflow/mlflow/blob/master/setup.py

Reply:
It is already included here : https://livegrep.dev.databricks.com/view/mlflow/mlflow/setup.py#L61. Do I need to add it in another place?

Reply:
Ah, sorry! I should've looked first. We'll still want to make this a required dependency (i.e., move it from `extras` to `install_requires`); otherwise, users will have to install `mlflow[extras]` to log artifacts on Databricks, which seems like a regression in the quality of our user experience. Let me know if that makes sense!

Reply:
That makes sense, done!